### PR TITLE
Use absolute paths so that coursera linters can be run anywhere in web

### PR DESCRIPTION
**Pull request by dennisli-coursera**

### CODE REVIEW

File: src/lint/linter/ArcanistCourserorgLinter.php [hunk 1]
```diff
@@ -42,18 +42,22 @@ public function getLintNameMap() {
   }
 
   public function lintPath($path) {
+    $absolutePath = realpath($path);
+    $webPath = $_SERVER['HOME'].'/base/coursera/web/';
+    $escapedWebPath = preg_quote($webPath, '/');
```
Comment (turadg) [inline src/lint/linter/ArcanistCourserorgLinter.php]: could you instead make a `$relPath` and use it in the existing preg_match calls?

Comment (dennisli-coursera) [inline src/lint/linter/ArcanistCourserorgLinter.php]: Done.

File: src/lint/linter/ArcanistCourserorgLinter.php [hunk 2]
```diff
@@ -42,9 +42,19 @@ public function getLintNameMap() {
   }
 
   public function lintPath($path) {
-    if (strstr($path, '/test/')) {
+    $absolutePath = realpath($path);
+    $webPath = $_SERVER['HOME'].'/base/coursera/web/';
```
Comment (turadg) [inline src/lint/linter/ArcanistCourserorgLinter.php]: this is constant throughout execution so should be at the top of the file.

**Comment by turadg** [comment]
One more suggestion, but feel free to land. Since this will make devs happier I don't want to delay it. ;-)

**Pull request merged**